ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -146,6 +152,16 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-additional-ca-policy
+- as: azure-ipi-f28-ota
```

Comment:
How about use arm arch to run the testing to save some cost.

Reply:
updated